### ISSUE TITLE
Stuck on loading initial ramdisk when booting 

### ISSUE THREAD

**Issue by GrieferMan** [instantOS/instantOS#150]
I installed instant os three times and I can't manage to boot it, every time it shows

Loading Linux linux-lts...
Loading initial ramdisk...

I don't know what to do,

**Comment by paperbenni** [comment]
What is the hardware you are trying to install on? Anything particularly proprietary

**Comment by GrieferMan** [comment]
> What is the hardware you are trying to install on? Anything particularly proprietary 

Yeah I have an nvidia gpu

**Comment by GrieferMan** [comment]
Is my gpu creating the problem?

**Comment by GrieferMan** [comment]
I will try to install it with nouveau and see if it makes a difference

**Comment by GrieferMan** [comment]
That didn't work either, can you please respond or something

**Comment by con-f-use** [comment]
Github is usually not the place to do live tech-support. It's for code developing. Maybe someone on the [discord](https://discord.io/instantos) or [matrix](https://matrix.to/#/!oegFDcozfbMAMKyeIE:matrix.org?via=matrix.org&via=t2bot.io&via=tchncs.de) can help as fast as you'd like. I understand you're frustrated, but this is a hobby project one guy does in his free time. Fast, responsive tech-support is something companies employ a whole division of people for.

**Comment by GrieferMan** [comment]
Yeah sorry I didn't know that they had a discord server ty for the info
